src/hits/hits.ts

```diff
@@ -11,6 +11,18 @@ import { connectHitsWithInsights } from 'instantsearch.js/es/connectors';
 import { BaseWidget } from '../base-widget';
 import { NgAisInstantSearch } from '../instantsearch/instantsearch';
 
+export type HitsState = {
+  hits: HitsItem[];
+  results: {};
+};
+
+export type HitsItem = {
```

Comment:
The entire `Hit` type can be taken from the InstantSearch.js [`Hit` type](https://github.com/algolia/instantsearch.js/blob/49a5b64e42deec3453669d7dce788395cf898ba8/src/types/instantsearch.ts#L46-L71).

Reply:
Is there a way that can be exported already from InstantSearch.js without publishing the typings?

Reply:
I'm not too sure since we're still experimenting with the types in the InstantSearch.js repo. A good time might be when we split InstantSearch into multiple packages.

Reply:
That type a looks a bit complex and will be subject to changes. I'm fearing we'll end up maintaining it in too many places, especially if we did this for every widget -> we'll just end up with a lot of inconsistent types everywhere.

My opinion is that it's best to keep it minimal for now and add the strict necessary. (maybe 
At a later point we will publish the types from InstantSearch. There will be less work adapting this small types to vanilla's more complex types.

Reply:
See https://github.com/algolia/angular-instantsearch/pull/553#discussion_r284135258.

Reply:
I've updated the type in a common place so that it can be shared between Hits and InfiniteHits. The PR is here https://github.com/algolia/angular-instantsearch/pull/579.

As for this PR, I'll just set HitsItem to Hit.

Reply:
done here 9bc26bd